site/styles/_carousel.scss

```diff
@@ -131,6 +131,13 @@ s72-carousel {
     max-width: 930px;
     padding-bottom: 0;
 
+    /* stylelint-disable */
+    display: -webkit-box;
+    overflow: hidden;
+    -webkit-line-clamp: 3;
+    -webkit-box-orient: vertical;
```

Comment:
Do we care that `box-orient` is deprecated?

Reply:
I care, but MDN is the one reporting it deprecated, and yet [use it in their own example](https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp). 🤷